dojo/tools/scout_suite/parser.py

```diff
@@ -8,7 +7,7 @@
 
 
 class ScoutSuiteParser(object):
-    """"ScoutSuite Wiki: https://github.com/nccgroup/ScoutSuite/wiki"""
+    """ "ScoutSuite Wiki: https://github.com/nccgroup/ScoutSuite/wiki"""
```

Comment:
The `"` before `ScoutSuite` seems unnecessary

Reply:
I removed blamk space in commit https://github.com/DefectDojo/django-DefectDojo/pull/8319/commits/1053acfe10f196c6dc7c94ff932aaff110e8fa17